test/browser/column/resizable.test.tsx

```diff
@@ -240,3 +241,92 @@ test('should remeasure flex columns when resizing a column', async () => {
   await expect.element(grid).toHaveStyle({ gridTemplateColumns: '79.1406px 919.422px 919.438px' });
   expect(onColumnResize).toHaveBeenCalledOnce();
 });
+
+test('should use columnWidths and onColumnWidthsChange props when provided', async () => {
+  const onColumnWidthsChangeSpy = vi.fn();
+  const onColumnResizeSpy = vi.fn();
+
+  function TestGrid() {
+    const [columnWidths, setColumnWidths] = useState(
+      (): ColumnWidths =>
+        new Map([
```

Comment:
FYI the return type isn't 100% enough to make `new Map(...)` type safe, for example you can add random properties in the object
![image](https://github.com/user-attachments/assets/4f064463-e77a-47ff-8f47-fa0277d72169)
https://github.com/microsoft/TypeScript/issues/55023
Might be good to export a type for the `type/width` object

Reply:
Done